unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/ManifestTranslator.cs

```diff
@@ -0,0 +1,350 @@
+                y++;
+                if (y == manifest.project.rows)
+                {
+                    x++;
+                    y = CommonScriptableObjects.playerCoords.Get().y;
+                }
+            }
+
+            //We prepare the mappings to the scenes
+            Dictionary<string, string> contentDictionary = new Dictionary<string, string>();
+            
+            foreach (var sceneObject in assets)
+            {
+                foreach (var content in sceneObject.contents)
+                {
+                    if(!contentDictionary.ContainsKey(content.Key))
+                        contentDictionary.Add(content.Key,content.Value);    
+                }
+            }
+
+            //We add the mappings to the scene
+            BIWUtils.AddSceneMappings(contentDictionary, BIWUrlUtils.GetUrlSceneObjectContent(),parcelData);
+            
+            //The data is built so we set the data of the scene 
+            scene.SetData(parcelData);
+
+            // We iterate all the entities to create the entity in the scene
+            foreach (BuilderEntity builderEntity in manifest.scene.entities.Values)
+            {
+                var entity = scene.CreateEntity(builderEntity.id);
+
+                bool nameComponentFound = false;
+                // We iterate all the id of components in the entity, to add the component 
+                foreach (string idComponent in builderEntity.components)
+                {
+                    //This shouldn't happen, the component should be always in the scene, but just in case
+                    if (!manifest.scene.components.ContainsKey(idComponent))
+                        continue;
+
+                    // We get the component from the scene and create it in the entity
+                    BuilderComponent component = manifest.scene.components[idComponent];
+                    
+                    switch (component.type)
+                    {
+                        case "Transform":
+                            DCLTransform.Model model = JsonConvert.DeserializeObject<DCLTransform.Model>(component.data.ToString());
+                            EntityComponentsUtils.AddTransformComponent(scene, entity, model);
+                            break;
+                        
+                        case "GLTFShape":
+                            LoadableShape.Model gltfModel = JsonConvert.DeserializeObject<LoadableShape.Model>(component.data.ToString());
+                            EntityComponentsUtils.AddGLTFComponent(scene, entity, gltfModel, component.id);
+                            break;
+                        
+                        case "NFTShape":
+                            //Builder use a different way to load the NFT so we convert it to our system
+                            string url = JsonConvert.DeserializeObject<string>(component.data.ToString());
+                            string assedId = url.Replace("ethereum://","");
```

Comment:
Can we extract these kind of strings (`"ethereum://"`) into constants?

Reply:
Sure!